packages/jest-core/src/runJest.ts

```diff
@@ -120,23 +126,25 @@ export default (async function runJest({
   onComplete,
   failedTestsCache,
 }: {
-  globalConfig: GlobalConfig,
-  contexts: Array<Context>,
-  outputStream: stream$Writable | tty$WriteStream,
-  testWatcher: TestWatcher,
-  jestHooks?: JestHookEmitter,
-  startRun: (globalConfig: GlobalConfig) => *,
-  changedFilesPromise: ?ChangedFilesPromise,
-  onComplete: (testResults: AggregatedResult) => any,
-  failedTestsCache: ?FailedTestsCache,
+  globalConfig: Config.GlobalConfig;
+  contexts: Array<Context>;
+  outputStream: NodeJS.WritableStream;
+  testWatcher: TestWatcher;
+  jestHooks?: JestHookEmitter;
+  startRun: (globalConfig: Config.GlobalConfig) => unknown;
+  changedFilesPromise?: ChangedFilesPromise;
+  onComplete: (testResults: TestResult.AggregatedResult) => any;
```

Comment:
other `onComplete`s are `void`, can we change this

Reply:
yeah, good catch